public/app/angular/panel/specs/metrics_panel_ctrl.test.ts

```diff
@@ -2,18 +2,12 @@ jest.mock('app/core/core', () => ({}));
 jest.mock('app/core/config', () => {
   return {
     ...jest.requireActual('app/core/config'),
-    bootData: {
```

Comment:
Here and in other places, we need to remove the partial-setting of config properties, as the general assumption is that `bootData.settings` is defined. 

With the `plugin_loader` mock removed, we now end up calling the real logic in `public/app/core/trustedTypePolicies.ts`, so we need to fixup some tests that ended up with an undefined `settings` property